src/applications/common/schemaform/save-in-progress/FormStartControls.jsx

```diff
@@ -90,21 +101,37 @@ class FormStartControls extends React.Component {
   }
 }
 
+
 FormStartControls.propTypes = {
+  prestartCheck: PropTypes.func,
+  prestartForm: PropTypes.func,
   formId: PropTypes.string.isRequired,
   handleLoadPrefill: PropTypes.func,
   migrations: PropTypes.array,
   returnUrl: PropTypes.string,
   fetchInProgressForm: PropTypes.func.isRequired,
   removeInProgressForm: PropTypes.func.isRequired,
-  router: PropTypes.object.isRequired,
   formSaved: PropTypes.bool.isRequired,
   prefillAvailable: PropTypes.bool.isRequired,
   startPage: PropTypes.string.isRequired,
   startText: PropTypes.string,
   resumeOnly: PropTypes.bool
 };
 
-export default withRouter(FormStartControls);
+function mapStateToProps(state) {
```

Comment:
I worry a little about connecting this component, since it's not immediately obvious why you would do it for just this field and not others. The prestart state and actions are integrated into the SiP code, so I wonder if we should just add these to the intro selector functions that the parent of this component uses.

Reply:
Great point @jbalboni, will update.